docs/Release-Process.md

```diff
@@ -30,11 +30,11 @@ the kata-containers release version.
         go mod tidy
         ```
         in the [cloud-api-adaptor](../src/cloud-api-adaptor/) directory and [csi-wrapper](../src/csi-wrapper/) directory.
+1. The CoCo operator updates to use references to the other component releases and then releases itself
 1. cloud-api-adaptor releases with the following phases detailed below:
```

Comment:
nit: s/1/2

Reply:
Ignore it.. In preview the sequence is proper.

Reply:
Yeah, I'm lazy/smart? and just label them all as `1.` so that when we insert new items, or re-juggle the order, it's less work to update them

Reply:
The is super smart :-) .. Learnt a new way..